src/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  border: 1px solid #f3f3f3;
```

Comment:
Consider using a variable for the border color instead of a hardcoded value `#f3f3f3`. This will improve maintainability and consistency across your styles.